compiler/semexprs.nim

```diff
@@ -409,8 +409,10 @@ proc semIs(c: PContext, n: PNode, flags: TExprFlags): PNode =
   else:
     if lhsType.base.kind == tyNone:
       # this is a typedesc variable, leave for evals
+      debug lhsType
```

Comment:
oops

Reply:
This was intentional, in case the CI run triggered this code path. I'll just remove this branch.